### DISCUSSION TITLE
SU2-Dev Meeting

### DISCUSSION THREAD

**Discussion by NAnand-TUD** [su2code/SU2#1781] [Q&A] [answered]
Guys, I lost track of SU2 Dev meetings. On a couple of occasions, I was the only one in the meeting. Can you please update me what are the new timing for the Dev Meetings?

Maybe we can have a Public Google Calender or equivalent?

Thanks,
Nitish

**Comment by pcarruscag** [comment, marked as answer]
We meet today in 40min

**Reply by pcarruscag** [reply]
Then next week Tuesday at 10am Pacific, and then we alternate
